.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release to npm
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version number to tag and release'
+        required: true
+        type: string
```

Comment:
Can we utilize?:

```
  workflow_dispatch:
    inputs:
      release_type:
        description: Type of release. patch or minor (major if breaking)
        required: true
        type: choice
        default: patch
        options:
          - patch
          - minor
          - major
```

https://github.com/newrelic/node-newrelic-aws-sdk/blob/6836a4896cf98a57d7c1359c36332f78364de698/.github/workflows/prepare-release.yml#L4-L14